src/upgrade-compatibility-tool/install.md

```diff
@@ -0,0 +1,123 @@
+---
+group: software-update-guide
+title: Install
+ee_only: True
+functional_areas:
+  - Upgrade
+---
+
+The Upgrade Compatibility Tool ALPHA is a command line tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+## Workflow
+
+The following diagram shows the expected workflow when running the Upgrade Compatibility Tool:
+
+![Upgrade Compatibility Tool Diagram](img/mvp-diagram-v2.png)
+
+## Who is the Upgrade Compatibility Tool for?
+
+The following use case describes the typical process for a Magento partner to upgrade a client's Magento instance:
+
+1. A partner's Software Engineer downloads the Upgrade Compatibility Tool package from the [Magento repository](https://repo.magento.com/) and executes it during the beta phase of the newest Magento release. See the [Download the Upgrade Compatibility Tool]({{site.baseurl}}/upgrade-compatibility-tool/install.html) topic for more information.
```

Comment:
@svera are we talking about the name of the package in the Magento repo (repo.magento.com)?, because the links you see here refer to the docs folder in the devdocs repo, which we had to change the name so urls show proper naming convention